packages/dialtone-css/lib/build/less/components/empty-state.less

```diff
@@ -16,41 +16,46 @@
   align-items: center;
   justify-content: center;
   text-align: center;
-  margin: 0 auto;
+  width: 100%;
+  padding: var(--dt-space-600);
 
-  &--size-sm {
-    width: 50%;
-    padding: var(--dt-space-500);
+  &__icon{
+    color: var(--dt-color-foreground-muted);
```

Comment:
Let's change this to `var(--dt-color-foreground-tertiary)` for now. Unfortunately quite a few icons haven't been flattened into a single `<path>`, so the translucent `muted` color results in overlapping parts (attached image). 

I'll log an issue to identify and fix those.

<img width="851" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/f4b1fd5d-aecc-42fc-8381-90e338ace4b1">